model/godleyIcon.cc

```diff
@@ -357,8 +357,8 @@ namespace minsky
       {
         // top justification at bottom of icon if displayed, bottom justified otherwise
         v->rotation(90);
-        v->moveTo(x, y + v->y() - (variableDisplay? v->top(): v->bottom()));
-        x+=v->width();
+        v->moveTo(x,y + v->x() - (variableDisplay? v->left(): v->right()));  // need to use x after stock vars have been rotated. for tickets 1230/1230
```

Comment:
That doesn't make much sense. Rotation only affects the coordinates of the icon components relative to the item's origin, not the meaning of Item::x and Item::y.

Reply:
I know, but I think it is because the variables are positioned relative to the Godley icon and not the canvas, so using the y coordinates here led to the irregular spacing in ticket 1232.